### PR TITLE
выполнил тест,вроде

### PR DESCRIPTION
**Pull request by Edward9601**
не судите строго)

### CODE REVIEW

File: Dumbuya_Test1/main.py [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+if len(a) == len(b):
+    for i in range(len(a)):
+        if a[i] == b[i]:
+            print('Equal')
+        else:
+            print('Not equal')
+
+string = {'five': 5, 'thirteen': 13, 'two': 2, 'eleven': 11, 'seventeen': 17,
+          'two': 2, 'one': 1, 'thirteen': 13, 'ten': 10, 'four': 4, 'eight': 8,
+          'five': 5, 'nineteen': 19}
```
Comment (MikitaTsiarentsyeu) [inline Dumbuya_Test1/main.py:32]: OK, so you have this dictionary, but where is the original string you supposed to work with?

File: Dumbuya_Test1/main.py [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+values = string.values()
+no_dublicates = set(values)
+no_dublicates2 = list(no_dublicates)
+list_mult = [((x * y), (x + y)) for (x, y) in zip(no_dublicates2[:-1], no_dublicates2[1:])]
```
Comment (MikitaTsiarentsyeu) [inline Dumbuya_Test1/main.py:37]: not exactly correct, the operations must alternate for different elements, mult for first pair, sum for second and so on

**Review by MikitaTsiarentsyeu — CHANGES_REQUESTED** [review]
The idea with zip is good, but realization is not exactly correct, please rework it.

**Comment by Edward9601** [comment]
About the last thing, did you mean that it should multiply 1st and 2d index, then sum 2d and 3rd and so on?

File: Dumbuya_Test1/main.py [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+                   'one': 1, 'ten': 10, 'four': 4, 'eight': 8, 'nineteen': 19}
+str_to_num = set()
+for s in string:
+    str_to_num.add(dict_for_string[s])
+
+no_duplicates = list(str_to_num)
+print(no_duplicates)
+
+summery = []
+last_action = True
```
Comment (MikitaTsiarentsyeu) [inline Dumbuya_Test1/main.py:40]: nitpick: not very good name for a variable, someone could think that it's not a switch for every action, but a cycle breaker.

**Review by MikitaTsiarentsyeu — APPROVED** [review]
Good job, your score is 9.0

**Pull request merged**